TopAnalysis/data/combineHLT/fcncTriLepton/2016.yaml

```diff
@@ -1,3 +1,4 @@
+######changed by Taiwon setup(DoubleMu, DoubleEG)
```

Comment:
Could be better to backup this 2016.yaml to different name and have new version.